web-console/src/views/datasource-view/datasource-view.tsx

```diff
@@ -657,7 +665,7 @@ GROUP BY 1`);
         <Switch
           checked={showDisabled}
           label="Show disabled"
-          onChange={() => this.setState({ showDisabled: !showDisabled })}
+          onChange={() => this.getDisabled(showDisabled)}
```

Comment:
This function is called `getDisabled` but it looks like it toggles the disabled state